src/OpenTelemetry/Metrics/MetricReaderTemporalityPreference.cs

```diff
@@ -28,7 +28,7 @@ public enum MetricReaderTemporalityPreference
     Cumulative = 1,
 
     /// <summary>
-    /// All measurements that are monotnic in nature are aggregated using delta temporality.
+    /// All measurements that are monotonic in nature are aggregated using delta temporality.
```

Comment:
Be careful! else you'll lose the reputation as the go-to person for any English wording questions 🤣